src/scss/components/hero-banner/_hero-banner.scss

```diff
@@ -364,4 +364,10 @@
     }
   }
 
+  // Override for the homepage's image hero banner only.
+  @include grid-media-min('md') {
+    .su-masthead>section:last-of-type {
+      padding-top: 1.9rem;
+    }
```

Comment:
@jenbreese 

I see a bit of white space because of padding on this element. Is that okay? I don't see this in the current iteration. E.g., https://advising.stanford.edu/

<img width="1701" height="108" alt="Screenshot 2025-08-04 at 1 44 17 PM" src="https://github.com/user-attachments/assets/b8afcae4-cc03-4c35-bcda-e3bc10762368" />

Reply:
Actually, I just realized it because of the utility links. When I remove them, the white space goes away.

<img width="1415" height="319" alt="Screenshot 2025-08-04 at 1 48 29 PM" src="https://github.com/user-attachments/assets/409d39f3-a382-452c-a223-c6fd5454826f" />

Reply:
Maybe this is new bug for this theme?

Reply:
@mdyoung3 Can you double check? the lines above should fix that white line. 
![Uploading Screenshot 2025-08-06 at 8.34.31 AM.png…]()

Reply:
@jenbreese Looks good.